languages/c/ast/ast_c_build.ml

```diff
@@ -17,7 +17,7 @@ open Either_
 open Ast_cpp
 module A = Ast_c
 
-let logger = Logging.get_logger [ __MODULE__ ]
+let tags = Logs_.create_tags [ __MODULE__ ]
```

Comment:
What is the difference between Logs tags and Logs src?
I've seen logs calls "tagged" with different src.

Reply:
I didn't know about sources or what to do with them. The [documentation](https://erratique.ch/software/logs/doc/Logs/Src/index.html) says it's something like the current library or the module but I don't see the point in making it different from ordinary tags.